File: Estrutura de Dados Simples/Ex2.py
A = []
B = []
C = []

print("Digite os valores do vetor A:")
for i in range(20):
    valor = float(input(f"A[{i}]: "))
    A.append(valor)

print("\nDigite os valores do vetor B:")
for i in range(20):
    valor = float(input(f"B[{i}]: "))
    B.append(valor)

for i in range(20):
    C.append(A[i] - B[i])

print("\nVetor C (A - B):")
for i in range(20):
    print(f"C[{i}] = {C[i]}")

File: Estrutura de Dados Simples/Ex2.cpp
#include <iostream>
using namespace std;

int main() {
    double A[20], B[20], C[20];

    cout << "Digite os valores do vetor A:" << endl;
    for (int i = 0; i < 20; i++) {
        cout << "A[" << i << "]: ";
        cin >> A[i];
    }

    cout << "\nDigite os valores do vetor B:" << endl;
    for (int i = 0; i < 20; i++) {
        cout << "B[" << i << "]: ";
        cin >> B[i];
    }

    for (int i = 0; i < 20; i++) {
        C[i] = A[i] - B[i];
    }

    cout << "\nVetor C (A - B):" << endl;
    for (int i = 0; i < 20; i++) {
        cout << "C[" << i << "] = " << C[i] << endl;
    }

    return 0;
}